src/java.desktop/share/classes/java/awt/color/ICC_ProfileRGB.java

```diff
@@ -232,33 +203,32 @@ public float getGamma(int component) {
      *         component whose TRC you want to retrieve: {@code REDCOMPONENT},
      *         {@code GREENCOMPONENT}, or {@code BLUECOMPONENT}
      * @return a short array representing the TRC
+     * @throws IllegalArgumentException if the component is not
+     *         {@code REDCOMPONENT}, {@code GREENCOMPONENT}, or
+     *         {@code BLUECOMPONENT}
      * @throws ProfileDataException if the profile does not specify the
```

Comment:
You said no CSR is needed but you document a previously undocumented exception, so I think a CSR is warranted

Reply:
CSR is needed and linked from the JBS, and after a few attempts, it is referenced from this pull request a few comments above.

Reply:
Ok. There's a very weird sequence above of it being added, and you removing it and whatever so it is confusing.
I'll find the CSR and review / ok it.